code/datums/uplink/revolution.dm

```diff
@@ -0,0 +1,32 @@
+/datum/uplink_item/item/revolution
+	category = /datum/uplink_category/revolution
+
+/datum/uplink_item/item/revolution/armory
+	name = "Armory Dropper"
+	desc = "A device that can be used to drop in an armory-worth of guns. Can only be used outside station areas, unless emagged, which is hazardous."
+	item_cost = 20
```

Comment:
The price here needs to be upped severely to be in line with the rest of the syndie items.